### DISCUSSION TITLE
Problem in Installing IPOPT

### DISCUSSION THREAD

**Discussion by amirbg4001** [coin-or/Ipopt#610] [General]
I am following the instruction for installing IPOPT on : 

**https://ethz.ch/content/dam/ethz/special-interest/mavt/dynamic-systems-n-control/idsc-dam/Research_Onder/Downloads/IPOPT/IPOPT_InstallWinMatlab_V0p1.pdf**

however, on step 6 for downloading ipopt, SVN requires a username and password, and by putting the correct username and password, I get an authentication failure error. Anyone Can help?

**Comment by svigerske** [comment]
The subversion repos are not publicly accessible anymore because we moved to GitHub years ago.
Uptodate installation instructions are at https://coin-or.github.io/Ipopt/INSTALL.html.

If you really want to follow these instructions for the installation of an old Ipopt, then you can get the Ipopt source tarballs from https://www.coin-or.org/download/source/Ipopt/.

**Reply by amirbg4001** [reply]
Thanks! I have started with the mentioned instructions. Lapack is being installed by MSYS2 however when building  HSL or IPOPT it does not recognize LAPACK I get the error below: 
checking for LAPACK... no
configure: error: Required package LAPACK not found

Do you know how to resolve this problem

**Reply by svigerske** [reply]
No. Check the generated config.log for details on why LAPACK was not found and then react accordingly.
